examples/osgi-test-example-bnd/itest/bnd.bnd

```diff
@@ -0,0 +1,47 @@
+-runrequires.junit: \
+	bnd.identity;id='junit-platform-launcher',\
+	bnd.identity;id='junit-jupiter-engine'
+
+# Build dependencies 
+-buildpath: \
+	${junit},\
+	${mockito},\
+	${osgitest},\
+	osgi.annotation;version='7.0',\
+	osgi.core;version='6.0',\
+	osgi.cmpn;version='7.0',\
+	junit-jupiter-api;version='5.7.0'
+	
+javac.source: 11
+javac.target: 11
+
+# We need JUnit and Mockito to resolve the test cases at runtime. 
+# Other runtime dependencies should be added as necessary
+-runbundles: \
```

Comment:
Sorting like we do in maven example:

```
# This will help us keep -runbundles sorted
-runstartlevel: \
    order=sortbynameversion,\
    begin=-1
```

Reply:
inserted